yaml2ics.py

```diff
@@ -55,7 +55,8 @@ def event_from_yaml(event_yaml: dict) -> ics.Event:
             print('Error: must specify end date for repeating events', file=sys.stderr)
             sys.exit(-1)
 
-        event.end = d.get('end', None)
```

Comment:
This is supposed to use the specified end date, and use None if it is not specified.

Reply:
My hypothesis is that this doesn't just set an attribute, but adjusts some internal data to set the end to None.  Which implicitly somehow sets the duration to zero:
https://github.com/ics-py/ics-py/blob/c1715525423c449f71fd8f718a70b00e5875fed9/src/ics/event.py#L105-L107
so that setting the end to None is basically setting duration=0.  When I thought about it, should setting the end here be any different from the `event = ics.Event(**d)` initialization.

Hm, looking right before, it is testing for `until=`.  I think `until` in the rrule specifies the end date of the recurrence (immediately below).  And `end` on the event object is the end time of an individual occurrence (done at the top).  Could that the source of the confusion?

Reply:
Since the RRULE support isn't yet official, this is a bit of a hack, so your analysis seems sensible.  Let's give it a shot.

Reply:
thanks!  Let's hope it works for others, too.